lib/blocks.php

```diff
@@ -131,6 +131,38 @@ function gutenberg_serialize_block( $block ) {
 	return $content;
 }
 
+/**
+ * Given an array of parsed blocks, returns the css styles they require.
+ *
+ * @since 1.8.1
+ *
+ * @param array $blocks Parsed block array.
+ *
+ * @return string CSS style block or empty string
+ */
+function compute_styles( $blocks ) {
+	$map_attrs_styles = array(
+		'textColor'       => 'color',
+		'backgroundColor' => 'background-color',
+	);
+	$styles           = '';
```

Comment:
Minor: If we added a newline before this, we could avoid the awkward `=` alignment, and more accurately associate the `$styles` variable with the `foreach` block where its value is accumulated.